docker/Dockerfile

```diff
@@ -4,7 +4,15 @@ RUN apt-get update -y && \
     apt-get install -y software-properties-common && \
     add-apt-repository ppa:deadsnakes/ppa && \
     apt-get update -y && \
-    apt-get install -y python3.7 python3-pip
+    apt-get install -y python3.7 python3-pip && \
+    apt-get update -y && \
+    apt-get install -y wget unzip
```

Comment:
Do you think we can add these packages to line 4 instead of their own line?

Reply:
Oh yea lol, done!